Fails to build when cross-compiling for i686

Presumably it would also fail when compiling natively on i686, but I don't have a x32 bit machine to test on.

```
error[E0425]: cannot find value `I686` in this scope
  --> /home/joshua/.local/lib/cargo/target/i686-unknown-linux-gnu/debug/build/target-lexicon-7aff8d211fe90ef0/out/host.rs:10:40
   |
10 |     architecture: Architecture::X86_32(I686),
   |                                        ^^^^ not found in this scope
   |
help: possible candidate is found in another module, you can import it into scope
   |
1  | use crate::targets::X86_32Architecture::I686;
   |

error[E0425]: cannot find value `I686` in this scope
  --> /home/joshua/.local/lib/cargo/target/i686-unknown-linux-gnu/debug/build/target-lexicon-7aff8d211fe90ef0/out/host.rs:20:30
   |
20 |         Architecture::X86_32(I686)
   |                              ^^^^ not found in this scope
   |
help: possible candidate is found in another module, you can import it into scope
   |
1  | use crate::targets::X86_32Architecture::I686;
   |

error[E0425]: cannot find value `I686` in this scope
  --> /home/joshua/.local/lib/cargo/target/i686-unknown-linux-gnu/debug/build/target-lexicon-7aff8d211fe90ef0/out/host.rs:56:48
   |
56 |             architecture: Architecture::X86_32(I686),
   |                                                ^^^^ not found in this scope
   |
help: possible candidate is found in another module, you can import it into scope
   |
1  | use crate::targets::X86_32Architecture::I686;
```

--- a/build.rs
+++ b/build.rs
@@ -59,6 +59,16 @@ fn write_host_rs(mut out: File, triple: Triple) -> io::Result<()> {
     writeln!(out, "#[allow(unused_imports)]")?;
     writeln!(out, "use crate::ArmArchitecture::*;")?;
     writeln!(out, "#[allow(unused_imports)]")?;
+    writeln!(out, "use crate::X86_32Architecture::*;")?;
+    writeln!(out, "#[allow(unused_imports)]")?;
+    writeln!(out, "use crate::Mips32Architecture::*;")?;
+    writeln!(out, "#[allow(unused_imports)]")?;
+    writeln!(out, "use crate::Mips64Architecture::*;")?;
+    writeln!(out, "#[allow(unused_imports)]")?;
+    writeln!(out, "use crate::Riscv32Architecture::*;")?;
+    writeln!(out, "#[allow(unused_imports)]")?;
+    writeln!(out, "use crate::Riscv64Architecture::*;")?;
+    writeln!(out, "#[allow(unused_imports)]")?;
     writeln!(out, "use crate::CustomVendor;")?;
     writeln!(out)?;
     writeln!(out, "/// The `Triple` of the current host.")?;

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -31,7 +31,8 @@ pub use self::host::HOST;
 pub use self::parse_error::ParseError;
 pub use self::targets::{
     Aarch64Architecture, Architecture, ArmArchitecture, BinaryFormat, CustomVendor, Environment,
-    OperatingSystem, Vendor,
+    Mips32Architecture, Mips64Architecture, OperatingSystem, Riscv32Architecture,
+    Riscv64Architecture, Vendor, X86_32Architecture,
 };
 pub use self::triple::{CallingConvention, Endianness, PointerWidth, Triple};
 

